frontend/packages/core/src/card.tsx

```diff
@@ -51,23 +49,65 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+  margin: "0px",
+  width: "100%",
+});
+
+const StyledCardHeaderAvater = styled.div({
+  marginRight: "16px",
```

Comment:
Should we include sizing for the icon being 32x32?

Reply:
yep added 32x32 to the icon container, left a TODO to use the material ui Avatar component when I implement the figma designs

Reply:
jst noticed the icon container isn't consistent across the mocks. left a Q on figma to confirm